deltachat-ios/Controller/AccountSetup/Instand Onboarding/InstantOnboardingView.swift

```diff
@@ -144,16 +147,18 @@ class InstantOnboardingView: UIView {
 
         if buttonShouldBeEnabled {
             agreeButton.backgroundColor = .systemBlue
+            otherOptionsButton.setTitleColor(.systemBlue, for: .normal)
         } else {
             agreeButton.backgroundColor = UIColor.systemGray.withAlphaComponent(0.3)
+            otherOptionsButton.setTitleColor(UIColor.systemGray.withAlphaComponent(0.5), for: .normal)
```

Comment:
From what I understand, `otherOptionsButton` isn't disabled here. It's just the text color that gets set to a light gray, if a user doesn't give a name. Is this intended?

Reply:
yeah, this is not optimal, but still better than before imo, see last commit message